MatrixSDK/LocationSharing/MXLocationService.swift

```diff
@@ -193,6 +224,61 @@ public class MXLocationService: NSObject {
         self.getAllBeaconInfo(forUserId: myUserId, inRoomWithId: roomId, completion: completion)
     }
     
+    // MARK: - Beacon info summary
+    
+    /// Get all beacon info summaries in a room
+    /// - Parameters:
+    ///   - roomId: The room id of the room
+    /// - Returns: Room beacon info summaries
+    public func getBeaconInfoSummaries(inRoomWithId roomId: String) -> [MXBeaconInfoSummaryProtocol] {
+        return self.session.aggregations.beaconAggegations.getBeaconInfoSummaries(inRoomWithId: roomId)
```

Comment:
You should use this PR to fix the name of this class beaconAgg**R**egations

Reply:
Good catch